doc/api/http.md

```diff
@@ -2046,6 +2049,10 @@ changes:
   * `ServerResponse` {http.ServerResponse} Specifies the `ServerResponse` class
     to be used. Useful for extending the original `ServerResponse`. **Default:**
     `ServerResponse`.
+  * `insecureHTTPParser` {boolean} Optionally overrides the value of
```

Comment:
I know the wording mirrors the option below, but I found it confusing.

> {boolean} Use an insecure HTTP parser that accepts invalid HTTP headers when `true`. Using the insecure parser should be avoided, see [`--insecure-http-parser`][] for more information. **Default:** `false`

Reply:
@sam-github Done!